Plugins/ExaTrkX/src/TorchEdgeClassifier.cpp

```diff
@@ -25,14 +25,18 @@ TorchEdgeClassifier::TorchEdgeClassifier(const Config& cfg,
       m_device(torch::Device(torch::kCPU)) {
   c10::InferenceMode guard(true);
   m_deviceType = torch::cuda::is_available() ? torch::kCUDA : torch::kCPU;
-  if (m_deviceType == torch::kCUDA && cfg.deviceID >= 0 &&
-      static_cast<std::size_t>(cfg.deviceID) < torch::cuda::device_count()) {
-    ACTS_DEBUG("GPU device " << cfg.deviceID << " is being used.");
-    m_device = torch::Device(torch::kCUDA, cfg.deviceID);
+  if (m_deviceType == torch::kCPU) {
+    ACTS_INFO("Running on CPU...");
```

Comment:
Could we change this to `ACTS_DEBUG` as well, to be consistent with GPU and also to mimize the verbosity of ACTS in default mode.

Reply:
done!